bokeh/models/annotations.py

```diff
@@ -410,6 +411,10 @@ class ColorBar(Annotation):
         (i.e. `low=0`), the tick and tick labels won't be rendered.
     """)
 
+    extend = Enum(Extend, default = "neither", help="""
+    Whether the low_color and high_color should be included as an extension
```

Comment:
`low_color` and `high_color` need to be quoted in double backticks. Use period at the end.